packages/next/src/client/components/react-dev-overlay/_experimental/internal/styles/Base.tsx

```diff
@@ -35,9 +35,10 @@ export function Base() {
           --color-text-color-red-1: #ff5555;
           --color-text-background-red-1: #fff9f9;
 
-          --font-stack-monospace: 'SFMono-Regular', Consolas, 'Liberation Mono',
+          --font-stack-monospace: 'Geist Mono', Consolas, 'Liberation Mono',
```

Comment:
Where are we initializing the font so it's available here in the overlay?

Reply:
Good catch, thanks! 🙏